### PR TITLE
Add MATERIALIZED_VIEW as table type

### PR DESCRIPTION
**Pull request by Gaurangi94**

### CODE REVIEW

File: connector/src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndITSuite.scala [hunk 1]
```diff
@@ -542,6 +542,15 @@ class SparkBigQueryEndToEndITSuite extends FunSuite
     assert(df.schema == allTypesTable.schema)
   }
 
+  test("query materialized view") {
+    var df = spark.read.format("bigquery")
+      .option("table", "bigquery-public-data:ethereum_blockchain.live_logs")
+      .option("viewsEnabled", "true")
+      .option("viewMaterializationProject", "bigquery-public-data")
```
Comment (davidrabinowitz) [inline connector/src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndITSuite.scala:549]: Please change the project name to `System.getEnv("GOOGLE_CLOUD_PROJECT")`. The public data sets are read only

File: connector/src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndITSuite.scala [hunk 2]
```diff
@@ -542,6 +542,15 @@ class SparkBigQueryEndToEndITSuite extends FunSuite
...
+      .option("viewMaterializationDataset", "ethereum_blockchain")
```
Comment (davidrabinowitz) [inline connector/src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndITSuite.scala:550]: Please change the dataset to `testDataset` (created at `beforeAll()`)

**Comment by davidrabinowitz** [comment]
in the `cloudbuild/cloudbuild.yaml` file, add the following entry for the integration test step
```
env:
  - 'GOOGLE_CLOUD_PROJECT=${_GOOGLE_CLOUD_PROJECT}'
```
This is needed to provide the environment variable for the test

**Comment by Gaurangi94** [comment]
@davidrabinowitz  Integration tests are failing with the following error - Cause: com.google.api.client.googleapis.json.GoogleJsonResponseException: 403 Forbidden
[info] POST https://www.googleapis.com/bigquery/v2/projects/google.com:hadoop-cloud-dev/datasets
[info] {
[info]   "code" : 403,
[info]   "errors" : [ {
[info]     "domain" : "global",
[info]     "message" : "Access Denied: Project google.com:hadoop-cloud-dev: User does not have bigquery.datasets.create permission in project google.com:hadoop-cloud-dev.",
[info]     "reason" : "accessDenied"
[info]   } ],
[info]   "message" : "Access Denied: Project google.com:hadoop-cloud-dev: User does not have bigquery.datasets.create permission in project google.com:hadoop-cloud-dev.",
[info]   "status" : "PERMISSION_DENIED"
[info] }

Could you grant me require perissions?

**Pull request merged**